homeworks/igor.orel_igor0806/GamesProject/about.html

```diff
@@ -0,0 +1,342 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta
+            name="viewport"
+            content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+        >
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>About</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+
+    <body>
+
+        <!-- Header  -->
+        <header class="intro">
+            <div class="nav">
+                <div class="logo">
+                    <img src="Icon/logo.svg" alt="img">
+                </div>
+
+                <div class="nav-menu">
+                    <ul class="link_list">
+
+                        <li class="link">
+                            <a class="nav-link" href="./index.html">Games</a>
+                        </li>
+
+                        <li class="link">
+                            <a class="nav-link" href="./about.html">About</a>
+                        </li>
+
+                        <li class="link">
+                            <a class="nav-link" href="#">Login</a>
+                        </li>
+
+                    </ul>
+                </div>
+            </div>
+        </header>
+
+        <h2 class="title_about">Study project "Games"</h2>
+
+        <div class="card_cards">
+            <div class="card">
```

Comment:
Please add a bit of semantics.
Card -`article`
Rows of game description - `ul` > `li`
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/4a19e5ab-7067-429f-aaad-178cb88b07fe)